### ISSUE TITLE
Q&A w/ @geek on Individual Membership Candidacy

### ISSUE THREAD

**Issue by mikeal** [nodejs/board#23]
This thread is for asking @geek questions regarding [his run](http://jsgeek.com/posts/node-foundation-board-nomination.html) 
for the Node.js Foundation Board of Directors.

**Comment by jasnell** [comment]
(A) If elected, what do you envision would be three most important issues that need to be addressed and how would you go about advocating for / resolving those?

(B) In a single sentence, how would you describe the role of the person elected to the board?

**Comment by rosskukulinski** [comment]
Hello @geek!  Thank you for applying to be an Individual Member on the NodeJS board.  I read your application and I have three questions:
1. Do you think your experiences on the node advisory board will benefit you on the NodeJS board? How?
2. Its great to see you are visible at NodeJS events, but many/most Node users are unable to attend events due to financial/family/work reasons.  How do you propose reaching out to the NodeJS community that don't overlap with you at these in-person events?
3. What do you hope to accomplish as a member of the board?  Short term (3-6 months) and long-term (12-24)?

**Comment by junosuarez** [comment]
1. What do you see as the most significant obstacle to node.js's continued growth?
2. What do you think the node.js foundation can do to address your answer to #1?

**Comment by geek** [comment]
@jasnell, @rosskukulinski, and @jden, thank you for your thoughtful questions.

TL;DR vote for Wyatt

To help answer the questions let me start with a recap of where we were, where we
are today, and how I plan to contribute to our future growth.  

It wasn't very long ago that I was joining weekly Node.js core team calls with
only a handful of people.  We were just trying to keep the project afloat with
what little time we were afforded.  I wasn't a core maintainer and didn't have
much time per week to even dedicate to the cause.  This was a dark time,
contributors to Node.js were neglected and people were losing hope in the future
of the project.  As a result, contributions stagnated and the project slowed.

Luckily, our community is made up of bright people, who decided to fork Node.js
and to let contributors have push access.  This excited many, including myself,
and we learned the value in not only listening, but in helping to empower
others.   This is a lesson that we carry with us to this day.  It is also one of
the reasons why I am excited that there are board seats for community members.
It demonstrates that we all have a voice when it comes to Node.js!

Around this time an advisory board was formed to help understand what was
happening and how to move forward.  I joined this advisory board only at the
end, at a point when the need for a foundation was fully realized.

Think about the added costs to a company if the codebase had continued to
fragment and splinter.  There would be definite costs to adopt new features or
bug fixes, and choosing the wrong fork could have other consequences.
Fortunately, we staved off the potential schism.  The way forward was clear,
companies act as good stewards by paying into the foundation who in turn support
the continued growth of Node.js. Some of these companies, like nearForm and IBM,
are exceptional stewards who go a step beyond the norm and pay people to work
full time to help maintain Node.js.   

The current setup is near perfect.  We have companies helping to finance a board
who in turn invest in public relations and marketing to spread the good word of
Node.js.  We also have a technical committee who is able to be fully responsive
to the needs of contributors.  

There is, however, one problem, which the current election process must address.  
Even though the current board is made up of wonderful people, the people are all
primarily located on the West coast of the US and are primarily representatives
of large companies.  This is a wonderful fact in and of itself, but it does mean
that there is a potential for the current board to be out of touch with future
adopters of Node.js, like people outside of the US or people who work at small
companies.

#### Answers

@jden

> What do you see as the most significant obstacle to node.js's continued growth?

The reputation of JavaScript, lack of a proven track record in the enterprise,
lack of diversity on the technical committee.

> What do you think the node.js foundation can do to address your answer to #1?

Work to convince more companies to share their success with Node.js, to pay
developers to contribute to Node.js and open source in general.  The board can
also work to form an international advisory group made up of representatives of
companies who have paid to the foundation and are located outside of the US.
Hopefully, there will also be a point where the foundation has enough funding to
pay a non-white male to contribute to Node.js core and join the technical
committee.

@rosskukulinski

> Do you think your experiences on the node advisory board will benefit you on the NodeJS board? How?

It won't hurt that I have been on calls with some of the current board members.
However, I believe a greater benefit with having me on the board is that I
bridge multiple communities.  I helped to build Node.js adoption at a large
company on the west coast and left that to work at nearForm, which is located in
Ireland.  In other words, I am not an entire outsider who will be dismissed,
while at the same time I have a lot of international input I can bring to the
meetings.

> Its great to see you are visible at NodeJS events, but many/most Node users are unable to attend events due to financial/family/work reasons. How do you propose reaching out to the NodeJS community that don't overlap with you at these in-person events?

Absolutely, which is one of the reasons I am a strong supporter of meetups.  I
really want to see more international speakers at local meetups.  This usually
means that a speaker will be telecommuted in.  I really do see the Node.js
community as special in that we have the general mentality that by helping the
individual the whole is helped too.  

I also think we should, at some point, do a virtual conference thats free.  If
this does happen, it would be wonderful if we can have a new speakers track, to
give more people in the community opportunities to teach everyone else.

> What do you hope to accomplish as a member of the board? Short term (3-6 months) and long-term (12-24)?

Node.js has a unique opportunity to become the de facto choice of developers of
tomorrow. If we don't step up our game, in terms of marketing and evangelism,
there is a potential for our Node.js growth to plateau.  However, we have a
limited budget for marketing dollars, so our investments need to be well
calculated.   The new applications of tomorrow, the ones where a choice is
allowed on what language or framework to use, will likely involve services that
talk to some internet of things.  Additionally, these services of tomorrow will
likely be an attempt at a microservices architecture.  Node.js shines bright in
this space and we need to do a better job at messaging it.  

In the short term I want to help spread the good word of the power of Node.js by
way of case studies.

We also should evaluate whether or not we should add a certification for
Node.js. Is this something the community wants or needs?  I will work with the
other individual member to gather feedback from the community on what we want
and need.

Additionally, I will help to make sure we are spending money on increasing our
adoption of Node.js globally.

Long term, I want to propose and help to form an international advisory group of
foundation companies that are located outside of the US.  

I also want to pursue a joint venture with education boards to provide them with
teaching materials related to JavaScript and Node.js.  With the push for
programming to be taught in schools, there is no reason why those courses can't
include sections on using JavaScript and Node.js.  

We need to make sure that Node.js has sustained growth long into the future.

@jasnell

>  If elected, what do you envision would be three most important issues that need to be addressed and how would you go about advocating for / resolving those?

See the previous response to @rosskukulinski.  If you would like to hear more, I
am more than happy to expand!

> In a single sentence, how would you describe the role of the person elected to the board?

To help actualize the widespread adoption of Node.js.

**Comment by rvagg** [comment]
Thanks for the answers @geek, great to have you running for this position.

I will add that you're not quite correct in saying _"the people are all primarily located on the West coast of the US and are primarily representatives of large companies."_ As the board representative for the TSC I get to diversify that a little bit in that (a) I'm in Australia and (b) I work at a small company, NodeSource, although it's true that we are an Enterprise-focused company.

**Comment by geek** [comment]
@rvagg 

60% of the board live on the West coast.  What I said is accurate.

You are certainly the outlier in terms of being the only non-US board member.  Hopefully, with this election, we can add a member to the board to help represent Europe and the Midwest ;)

**Comment by mikeal** [comment]
Election is over, results are [posted](https://github.com/nodejs/membership/blob/master/README.md#2016-election).
